core/src/main/java/io/aiven/klaw/service/AclControllerService.java

```diff
@@ -1,10 +1,6 @@
 package io.aiven.klaw.service;
 
-import static io.aiven.klaw.model.enums.MailType.ACL_DELETE_REQUESTED;
-import static io.aiven.klaw.model.enums.MailType.ACL_REQUESTED;
-import static io.aiven.klaw.model.enums.MailType.ACL_REQUEST_APPROVED;
-import static io.aiven.klaw.model.enums.MailType.ACL_REQUEST_DENIED;
-import static io.aiven.klaw.model.enums.MailType.ACL_REQUEST_FAILURE;
+import static io.aiven.klaw.model.enums.MailType.*;
 import static org.springframework.beans.BeanUtils.copyProperties;
```

Comment:
This .* snuck in.

Reply:
Fixed. Intellij settings. It should be ok now.